cpplint.py

```diff
@@ -0,0 +1,6244 @@
+#!/usr/bin/env python
```

Comment:
Um. What's this doing here?!

Reply:
As in, `cpplint.py` shouldn't be in our library

Reply:
sorry